src/elements/navigation/common/navigation-action-common.ts

```diff
@@ -21,7 +21,7 @@ export declare class SbbNavigationActionCommonElementMixinType {
   public accessor size: SbbNavigationActionSize;
   public get marker(): SbbNavigationMarkerElement | null;
   public get section(): SbbNavigationSectionElement | null;
-  public connectedSection: SbbNavigationSectionElement | null;
+  public connectedSection: SbbNavigationSectionElement | undefined;
```

Comment:
What was the problem with null?

Reply:
On the manifest the type is `SbbNavigationSectionElement | undefined` because is taken from the mixin getter, while the generated component has null because uses the MixinType;
so when i create the setter in Angular I have a type mismatch

Reply:
I'd suggest to keep null, but change
`public connectedSection?: SbbNavigationSectionElement` to
`public connectedSection: SbbNavigationSectionElement | null = null;`
would that work?

Reply:
Careful. As this is not a `@property`, it will not be retained during an upgrade, meaning the `= null` could overwrite a previously set value.